setup.py

```diff
@@ -33,14 +37,25 @@
     license="MIT",
     classifiers=[
         "License :: OSI Approved :: MIT License",
+        "Intended Audience :: Developers",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3 :: Only",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
     ],
-    packages=find_packages(),
+    python_requires=">=3.7",
+    packages=find_packages(
+        include=["stytch*"],
+        exclude=["*.test", "*.tests", "*.test.*", "*.tests.*"],
+    ),
```

Comment:
Just to confirm because I don't understand Python packaging enough: This is what ensures that we only package the actual `stytch` library and not any of our tests or codegen (or codegen tests), right?

Reply:
Correct, the codegen and tests aren't relevant to put in the PyPI package